queries.sql

```diff
@@ -41,3 +41,63 @@ SELECT *
 SELECT *
     FROM animals
     WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+-- MILESTONE 2
+-- Inside a transaction update the animals table by setting the species column to unspecified. 
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+    UPDATE animals SET species = 'unespecified';
+    ROLLBACK;
+COMMIT;
```

Comment:
- [ ] On this transaction, Please verify that the change was made. Hint: `SELECT species from animals;`.